src/app/components/MediaLoader/configs/tvMedia.ts

```diff
@@ -0,0 +1,49 @@
+import filterForBlockType from '#lib/utilities/blockHandlers';
+import { ConfigBuilderProps, ConfigBuilderReturnProps } from '../types';
+import buildPlaceholderConfig from '../utils/buildPlaceholderConfig';
+
+export default ({
+  blocks,
+  basePlayerConfig,
+  translations,
+}: ConfigBuilderProps): ConfigBuilderReturnProps => {
+  const { model: tvMediaBlock } = filterForBlockType(blocks, 'tvMedia');
+
+  const placeholderConfig = buildPlaceholderConfig({
+    title: tvMediaBlock.episodeTitle,
+    duration: tvMediaBlock.versions[0].duration,
+    durationISO8601: tvMediaBlock.versions[0].durationISO8601,
+    type: 'video',
+    holdingImageURL: `https://${tvMediaBlock.imageUrl}`,
+    placeholderImageLocator: `https://${tvMediaBlock.imageUrl}`,
+    placeholderImageOriginCode: 'pips',
+    translations,
+  });
+
+  return {
+    playerConfig: {
+      ...basePlayerConfig,
+      autoplay: false,
+      statsObject: {
+        ...basePlayerConfig.statsObject,
+        episodePID: tvMediaBlock.id,
+      },
+      playlistObject: {
+        title: tvMediaBlock.episodeTitle,
+        holdingImageURL: `https://${tvMediaBlock.imageUrl}`,
+        items: [
+          {
+            versionID: tvMediaBlock.versions[0].versionId,
+            kind: tvMediaBlock.smpKind,
+            duration: tvMediaBlock.versions[0].duration,
+            vpid: tvMediaBlock.versions[0].versionId,
+          },
+        ],
+        summary: tvMediaBlock.synopses.short,
+      },
```

Comment:
As per the aresMedia/clipMedia, if embedding is true, then we need to set embedRights to allowed:   https://github.com/bbc/simorgh/blob/227b97d6bbb133584e84dc75b7e7987089989438/src/app/components/MediaLoader/configs/clipMedia.ts#L101-L102

```suggestion
        summary: tvMediaBlock.synopses.short,
        ...(tvMedia.embedding && { embedRights: 'allowed' })
      },
```

Reply:
Are we wanting `tvMediaBlock` instead of `tvMedia` here?

Reply:
Oops yes, copy paste error 😄